apps/backend/src/modules/project-groups/project-group.controller.ts

```diff
@@ -81,7 +87,13 @@ class ProjectGroupController extends BaseController {
 				this.delete(options as APIHandlerOptions<{ params: { id: string } }>),
 			method: "DELETE",
 			path: ProjectGroupsApiPath.$ID,
-			preHandlers: [checkUserPermissions([PermissionKey.MANAGE_USER_ACCESS])],
+			preHandlers: [
+				checkUserPermissions([
+					PermissionKey.MANAGE_PROJECT,
+					PermissionKey.VIEW_ALL_PROJECTS,
```

Comment:
User with only `View All Projects` permissong shouldn't be able to delete project group